pkg/workloadapi/workload.proto

```diff
@@ -267,6 +268,11 @@ message Workload {
   // Extension provides a mechanism to attach arbitrary additional configuration to an object.
   repeated Extension extensions = 26;
 
+  // Capacity for this workload.
+  // This represents the amount of traffic the workload can handle, relative to other workloads
+  // If unset, the capacity is default to 1.
+  google.protobuf.UInt32Value capacity = 27;
```

Comment:
Does it indicate how much traffic **should** be sent to this workload? What does 1 mean here?

Reply:
Could be misunderstanding but I think 1 means nothing in absolute terms. The value is only meaningful in context when used to create a ratio. 1 just acts as a better default than 0 since 0 probably could have an absolute meaning which is probably not suitable (I want no traffic at all?) and stinks for creating ratios (assuming you do want at least some traffic going to all WL).

So If you have 5 workloads set to 1 they should each take roughly equal traffic. When you change a workload to be capacity 2 then you should get 4 WL each handling 1/6th of the traffic and the final WL takes 1/3. Conversely I guess you could set the final WL to 0 and get 4 WL each handling a quarter of the traffic.

Reply:
It seems to me that it is "weight" of how much traffic ztunnel should send to each workload? Curious why it is called "Capacity" not "weight"?

Reply:
Or `capacity_weight` - I agree it's a bit unintuitive.

Reply:
Yeah, FWIW, I didn't mean to close the PR; I thought another review was still going to be needed. Happy to continue to iterate on the name

Reply:
The naming was pretty intentional from Costin and Louis. I don't fully grok it or care too much but my understand is weight is basically an imperative statement of "you should send exactly x% of traffic" while capacity is a declarative statement of "this endpoint can handle this much traffic" which is more flexible. Google does a lot of "capacity" load balancing (https://cloud.google.com/load-balancing/docs/tutorials/about-capacity-optimization-with-global-lb etc).

In its current state we treat them the same anyways, so the naming is more about future proofing